taichi/program/async_engine.cpp

```diff
@@ -188,15 +210,13 @@ void ExecutionQueue::enqueue(TaskLaunchRecord &&ker) {
       auto func = codegen->codegen();
       async_func->set(func);
     });
+    trash_bin.push_back(std::move(cloned_stmt));
```

Comment:
This is `ExecutionQueue::trash_bin`. Ideally we should have only one trash bin in `IRBank`, but `ExecutionQueue` can't access `IRBank` directly...

Reply:
Hmm, could you then pass IRBank from `AsyncEngine` to `ExecutionQueue`?